docs/home.md

```diff
@@ -7,113 +7,67 @@
-   ovms_docs_model_cache
-   ovms_docs_metrics
-   ovms_sample_cpu_extension
-   ovms_docs_dynamic_input
    ovms_docs_stateful_models
-   ovms_docs_custom_loader
+   ovms_docs_binary_input
    ovms_docs_performance_tuning
-   ovms_docs_kubernetes
+   ovms_docs_advanced
    ovms_docs_demos
    ovms_docs_troubleshooting
 
 @endsphinxdirective
 
+OpenVINO&trade; Model Server (OVMS) is a high-performance system for serving models. Implemented in C++ for scalability and optimized for deployment on Intel architectures. OVMS uses the same architecture and API as [TensorFlow Serving](https://github.com/tensorflow/serving) and [KServe](https://github.com/kserve/kserve) while applying OpenVINO for inference execution. Inference service is provided via gRPC or REST API, making deploying new algorithms and AI experiments easy.
+
+![OVMS diagram](ovms_draft_diagram.png)
+
+## Model Serving
+
+A model server is a web server that hosts models and makes them accessible to software components over standart network protocols. Functionally it works similarly to a web server: you send a request and receive a response. Just like a web server, the model can be accessed across devices as long as they are connected via a common network.
 
 ![OVMS picture](ovms.png)
 
-OpenVINO&trade; Model Server (OVMS) is a high-performance system for serving machine learning models. It is based on C++ for high scalability 
-and optimized for Intel solutions, so that you can take advantage of all the power of the Intel® Xeon® processor or Intel’s AI accelerators 
-and expose it over a network interface. OVMS uses the same architecture and API as [TensorFlow Serving](https://github.com/tensorflow/serving), 
-while applying OpenVINO for inference execution. Inference service is provided via gRPC or REST API, making it easy to deploy new algorithms and AI experiments.
+The key advantage of a model server is its ability to process several client requests simultaneously so that the model uses less device memory. 
+As a result, you can create a lightweight client with only the necessary functions to perform API calls for edge or cloud deployments. 
 
-Model repositories may reside on a locally accessible file system (e.g. NFS), as well as online storage compatible with 
-Google Cloud Storage (GCS), Amazon S3, or Azure Blob Storage. 
+## OpenVINO Model Server Key Features: 
 
-Read [release notes](https://github.com/openvinotoolkit/model_server/releases) to find out what’s new.
+The models used by OpenVINO Model Server need to be stored locally or hosted remotely by object storage services. For more details, refer to [Preparing Model Storage](./models_repository.md) documentation.  
 
-Review the [Architecture concept](architecture.md) document for more details.
+OpenVINO&trade; Model Server works with [Bare Metal Hosts](host.md) as well as [Docker containers](docker_container.md). It is also suitable for landing in the [Kubernetes environment](../deploy/README.md).
 
-Key features: 
```

Comment:
It's good to make some structure in presenting features here, also we could probably add API Reference Guide?
My first guess for structure is Basic/Advanced feature, but maybe you have a better idea (eg Model/Server/Client... or according to what users would need to use first, etc)